shell/platform/tizen/tizen_embedder_engine.h

```diff
@@ -83,6 +83,7 @@ class TizenEmbedderEngine {
   void AppIsPaused();
   void AppIsDetached();
 
+  void SendWindowMetrics(int32_t width, int32_t height, double pixel_ratio);
```

Comment:
This line should come before the line 79 so that it matches with the implementation order.
```cpp
  void SendWindowMetrics(int32_t width, int32_t height, double pixel_ratio);
  void SetWindowOrientation(int32_t degree);
```

Reply:
I fixed it